lachesis-rs/src/server/handlers.rs

```diff
@@ -0,0 +1,90 @@
+use actix_web::{http, AsyncResponder, Error, HttpMessage, HttpRequest, HttpResponse};
+
+use futures::{future::result, Future};
+
+use super::AppState;
+
+use super::heartbeat::GetHeartbeatCount;
+
+use actix::prelude::*;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct SubmitTransaction {
+    pub signature: String,
+    pub payload: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct CheckTransactionStatus {
+    id: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Peer {
+    id: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct PeerList {
+    peers: Vec<Peer>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub enum TransactionStatus {
+    Complete,
+    Pending,
+    Failed,
+}
+
+pub fn submit_transaction(
+    req: &HttpRequest<AppState>,
+) -> Box<Future<Item = HttpResponse, Error = Error>> {
+    req.json()
+        .from_err()
+        .and_then(|val: SubmitTransaction| {
+            println!("model: {:?}", val);
+            Ok(HttpResponse::Ok().json(val)) // <- send response
+        })
+        .responder()
+}
+
+pub fn heartbeat(req: &HttpRequest<AppState>) -> HttpResponse {
+    println!("{:?}", req);
```

Comment:
Do we need this line in production, or it is a debugging leftover?

Reply:
debugging left over, needs to be removed